docs/explanation/index.rst

```diff
@@ -0,0 +1,12 @@
+============
+Explanations
+============
+
+Explanations in this section will discus various topics and concepts
```

Comment:
Repeating the heading here feels unnatural, perhaps "In this section we will ..."